examples/catalyst_contrastive/callbacks.py

```diff
@@ -0,0 +1,76 @@
+from typing import Any, Callable, Dict, List, Tuple, Union
+from collections import Counter
+from functools import partial
+from pathlib import Path
+import pickle
+
+import torch
+
+from catalyst.callbacks.metric import LoaderMetricCallback
+from catalyst.contrib.nn.modules import se
+from catalyst.core import Callback, CallbackOrder, IRunner
+from catalyst.core.callback import ICallback
+from catalyst.metrics._metric import AccumulationMetric, ICallbackLoaderMetric
+from catalyst.registry import REGISTRY
+
+
+class SklearnClassifierCallback(ICallback):
```

Comment:
btw, I think it could be a valuable addition for catalyst callbacks overall ;)

Reply:
Hmm.. Can I do this in [this way](https://github.com/catalyst-team/catalyst/pull/1261/commits/2a666d32c841a69a82335337855304ecaeea68b3)?